README.md

```diff
@@ -1,17 +1,15 @@
 # Terragrunt
 
-Terragrunt is a thin wrapper for [Terraform](https://www.terraform.io/) that provides extra tools for working with
-multiple Terraform modules, remote state, and locking.
-
-
+Terragrunt is a thin wrapper for [Terraform](https://www.terraform.io/) that provides extra tools for keeping your Terraform scripts DRY, working with multiple Terraform modules, and managing remote state.
```

Comment:
Done.

Good catch. I see that mentioned in the Terraform docs [here](https://www.terraform.io/docs/configuration/index.html).